resources/domain/transaction/index.ts

```diff
@@ -13,8 +13,12 @@ export interface TransactionData extends Omit<JsonTx, 'chainId' | 'type'> {
   chainId: string
   type: string
   gasFeesSource: GasFeesSource
+  recipientType?: string
 }
 
+export function isNonZeroHex(hex: string) {
```

Comment:
I think we can move this function to a utils module or something as its not transaction-specific